app/controllers/manage_consents_controller.rb

```diff
@@ -68,6 +68,28 @@ def current_step
     wizard_value(step).to_sym
   end
 
+  def finish_wizard_path
+    redirect_path =
+      case @route
+      when "consents"
+        consents_session_path(@session)
+      when "triage"
+        triage_session_path(@session)
+      else
+        vaccinations_session_path(@session)
+      end
+
+    flash[:success] = {
+      heading: "Record saved for #{@patient.full_name}",
+      body:
+        ActionController::Base.helpers.link_to(
+          "View child record",
+          session_patient_triage_path(@session, @patient)
+        )
+    }
+    redirect_path
+  end
```

Comment:
If you're revisiting this code at this point, consider (if you haven't) moving the `case ... when` to the bottom of the method and just returning it's result. `redirect_path` doesn't get used in the flash.